src/helpers/augmentedReality/multipleSceneIndexGenerator.ts

```diff
@@ -24,36 +42,73 @@ export const multipleSceneIndexGenerator = ({
   let modelIndex = 0;
   const scenesCount = scenes?.length;
   let localUris = scenes?.[modelIndex]?.localUris;
+  const models: TModels = [];
+  const textures: TTextures = [];
 
   // if we have multiple scenes, we want to calculate the model and texture based on the current day
   // compared to the `startDate`
   if (scenesCount > 1 && startDate && timePeriodInDays) {
     // all models must have the same number of variable textures. therefore, in order to obtain the
     // number of textures, the textures of the first model were calculated.
-    const variableTextures = scenes[0]?.localUris?.filter(
+    let variableTextures = scenes[0]?.localUris?.filter(
       ({ type, stable }) => type === 'texture' && !stable
     );
     const variableTexturesCount = variableTextures?.length;
 
-    if (!variableTexturesCount) {
-      // nothing to do if there are no unstable textures
-      // keep original localUris with modelIndex = 0
-      return { localUris };
-    }
-
```

Comment:
what if `variableTexturesCount` is zero now?

Reply:
I think we're gonna run into error right now. Now we need models and textures arrays inside localUris. I'll think of a new logic for this.